rclcpp_lifecycle/test/benchmark/benchmark_lifecycle_node.cpp

```diff
@@ -162,18 +162,18 @@ BENCHMARK_F(BenchmarkLifecycleNode, transition_valid_state)(benchmark::State & s
   reset_heap_counters();
   for (auto _ : state) {
     (void)_;
-    const auto & active =
+    const rclcpp_lifecycle::State & active =
       node->trigger_transition(lifecycle_msgs::msg::Transition::TRANSITION_ACTIVATE);
     if (active.id() != lifecycle_msgs::msg::State::PRIMARY_STATE_ACTIVE) {
       state.SkipWithError("Transition to active state failed");
     }
-    const auto & inactive =
+    const rclcpp_lifecycle::State & inactive =
       node->trigger_transition(lifecycle_msgs::msg::Transition::TRANSITION_DEACTIVATE);
     if (inactive.id() != lifecycle_msgs::msg::State::PRIMARY_STATE_INACTIVE) {
       state.SkipWithError("Transition to inactive state failed");
     }
-    benchmark::DoNotOptimize(active);
-    benchmark::DoNotOptimize(inactive);
+    benchmark::DoNotOptimize(const_cast<rclcpp_lifecycle::State &>(active));
```

Comment:
Might want a comment as to why you're doing this, if I saw a `const_cast` like this, I'd be very curious, or maybe just drop it if I was editing this code.

Reply:
Yeah, that is an excellent point.  I'll go ahead and add in some comments.

Reply:
Fixed in 2af0448.